### PR TITLE
MAUI-5256 : Prepare styles UG for .NET MAUI Popup

### PR DESCRIPTION
**Pull request by RiyashameedM**
Prepared styles UG for .NET MAUI Popup

### CODE REVIEW

File: MAUI/Popup/styles.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+The SfPopup allows customizing the header elements with various header customizations as follows:
+
+<table>
+<tr>
+<th> Property </th>
+<th> Description </th>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.HeaderBackground </td>
```
Comment (PiruthivirajM) [inline MAUI/Popup/styles.md:24]: remove "SfPopup.PopupStyle.

File: MAUI/Popup/styles.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with header customization](/Images/styles/maui-popup-with-header-customization.png)
+
+## Styling popup footer
+
+The SfPopup allows customizing the footer elements with various footer customizations as follows:
```
Comment (PiruthivirajM) [inline MAUI/Popup/styles.md:87]: The SfPopup allows customizing the footer appearance using the below properties.

File: MAUI/Popup/styles.md [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+documentation: ug
+--- 
+
+# Styles in .NET MAUI Popup (SfPopup)
+
+The SfPopup applies style to all of its elements by using the `SfPopup.PopupStyle` property.
+
+## Styling popup header
+
+The SfPopup allows customizing the header elements with various header customizations as follows:
```
Comment (PiruthivirajM) [inline MAUI/Popup/styles.md:16]: The SfPopup allows customizing the header appearance using the below properties.

File: MAUI/Popup/styles.md [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+
+The SfPopup allows customizing the stroke appearance with various stroke customizations as follows:
+
+<table>
+<tr>
+<th> Property </th>
+<th> Description </th>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.Stroke </td>
```
Comment (PiruthivirajM) [inline MAUI/Popup/styles.md:166]: remove "SfPopup.PopupStyle

File: maui-toc.html [hunk 5]
```diff
@@ -669,6 +672,6 @@
...
....2.0.36">v20.2.0.36 Main release</a></li></ul></li><li>2022 volume 1 - v20.1.0.*<ul><li> Weekly Nuget Release <ul><li><a href="/maui/release-notes/v20.1.0.56">v20.1.0.56</a></li><li><a href="/maui/release-notes/v20.1.0.52">v20.1.0.52</a></li><li><a href="/maui/release-notes/v20.1.0.51">v20.1.0.51</a></li><li><a href="/maui/release-notes/v20.1.0.50">v20.1.0.50</a></li><li><a href="/maui/release-notes/v20.1.0.48">v20.1.0.48</a></li></ul></li><li><a href="/maui/release-notes/v20.1.0.55">v20.1.0.55 Service Pack Release</a></li><li><a href="/maui/release-notes/v20.1.0.47">v20.1.0.47 Main Release</a></li></ul></li><li>2021 Volume 4  - v19.4.0.*<ul><li><a href="/maui/release-notes/v19.4.0.48">v19.4.0.48 Service Pack Release</a></li><li><a href="/maui/release-notes/v19.4.0.38">v19.4.0.38 Main Release</a></li></ul></li><li>2021 volume 3 - v19.3.0.*<ul><li> Weekly Nuget Release <ul><li><a href="/maui/release-notes/v19.3.0.45">v19.3.0.45</a></li><li><a href="/maui/release-notes/v19.3.0.44">v19.3.0.44</a></li></ul></li><li><a href="/maui/release-notes/v19.3.0.53">v19.3.0.53 Service Pack Release</a></li><li><a href="/maui/release-notes/v19.3.0.43">v19.3.0.43 Main Release</a></li></ul></li></ul>
```
Comment (PiruthivirajM) [inline maui-toc.html:672]: avoid unwanted changes

File: MAUI/Popup/styles.md (hunk 1 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:24]: modified

File: MAUI/Popup/styles.md [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+
+The SfPopup allows customizing the footer elements with various footer customizations as follows:
+
+<table>
+<tr>
+<th> Property </th>
+<th> Description </th>
+</tr>
+<tr>
+<td> SfPopup.PopupStyle.FooterBackground </td>
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:95]: modified

File: MAUI/Popup/styles.md [hunk 7]
```diff
@@ -0,0 +1,299 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with footer customization](/Images/styles/naui-popup-with-footer-customization.png)
+
+## Stroke customization
+
+The SfPopup allows customizing the stroke appearance with various stroke customizations as follows:
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:158]: resolved

File: MAUI/Popup/styles.md (hunk 2 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:87]: modified

File: MAUI/Popup/styles.md (hunk 4 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:166]: modified

File: MAUI/Popup/styles.md [hunk 8]
```diff
@@ -0,0 +1,299 @@
...
+        <sfPopup:SfPopup x:Name="popup">
+            <sfPopup:SfPopup.PopupStyle>
+                    <sfPopup:PopupStyle OverlayColor="LightPink" />
+            </sfPopup:Sfpopup.PopupStyle>
+        </sfPopup:SfPopup>
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    popup.PopupStyle.OverlayColor = Colors.LightPink;
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/styles.md:227]: modified

**Pull request closed without merging**